ee/tuf/generator/generator.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/logutil"
+	"github.com/theupdateframework/go-tuf/client"
+	filejsonstore "github.com/theupdateframework/go-tuf/client/filejsonstore"
+)
+
+// initialRootJSON contains the trusted first version of root.json
+// This is version 1 from https://tuf.kolide.com/repository/1.root.json
+var initialRootJSON = []byte(`{"signed":{"_type":"root","spec_version":"1.0","version":1,"expires":"2023-03-29T15:57:20-04:00","keys":{"53a3eb57d9f5e755ec73e95429033b78423c9c01cef8c8d4e9911b6fe01198c8":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"344ca850ae47f65f6ffd6e9e6268524a237401000dcb892b9ddad82ca0083a65"}},"bcd5f36e7a357e7fba38b6b7c531f7fcf509eb2da2cfcdfa94bce99ad18a7317":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"8adc59a24b5c044794e002002792a4306bb63337d1f2cec961dbe68ccafdc874"}},"bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"aea7938b3b8f9cc0fc910899beb6a025487db87d6c2ea9a12fdf54c25817c814"}},"e434c83555ba93337e91c073095a164e4ae66f01a91cf5e8d4228f2382568d19":{"keytype":"ed25519","scheme":"ed25519","keyid_hash_algorithms":["sha256","sha512"],"keyval":{"public":"785323acb700ba835da550c1ac56300cf87890b4367c7cb1c14c1299e5e37075"}}},"roles":{"root":{"keyids":["bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508"],"threshold":1},"snapshot":{"keyids":["bcd5f36e7a357e7fba38b6b7c531f7fcf509eb2da2cfcdfa94bce99ad18a7317"],"threshold":1},"targets":{"keyids":["e434c83555ba93337e91c073095a164e4ae66f01a91cf5e8d4228f2382568d19"],"threshold":1},"timestamp":{"keyids":["53a3eb57d9f5e755ec73e95429033b78423c9c01cef8c8d4e9911b6fe01198c8"],"threshold":1}},"consistent_snapshot":true},"signatures":[{"keyid":"bd9bbc72f9a1c7cd5998c94b5a31fedb997315f81758f90dbe393743ec984508","sig":"b0624815cf3b8749edc68c96ffca588ce532e9cdf8ac4db4748e260c186ecb3f975d555cf25fe5ddba418f2d3d2af4e5c2a2046b20619ddfd30f3390560f850d"}]}`)
```

Comment:
I would put this into an `assets` folder somewhere, and not inline it here. The problem with inlineing it, is that it makes it ugly when we update it.

Reply:
This is version 1, so we shouldn't ever update it